core/src/main/java/org/axonframework/commandhandling/AsynchronousCommandBus.java

```diff
@@ -47,39 +51,39 @@
     private final Executor executor;
 
     /**
-     * Initialize the AsynchronousCommandBus, using a Cached Thread Pool.
-     */
-    public AsynchronousCommandBus() {
-        this(Executors.newCachedThreadPool());
-    }
-
-    /**
-     * Initialize the AsynchronousCommandBus using the given {@code executor}.
+     * Instantiate a {@link AsynchronousCommandBus} based on the fields contained in the {@link Builder}.
+     * <p>
+     * Will assert that the {@link TransactionManager}, {@link MessageMonitor}, {@link RollbackConfiguration} and
+     * {@link Executor} are not {@code null}, and will throw an {@link AxonConfigurationException} if any of them is
+     * {@code null}.
      *
-     * @param executor The executor that processes Command dispatching threads
+     * @param builder the {@link Builder} used to instantiate a {@link AsynchronousCommandBus} instance
      */
-    public AsynchronousCommandBus(Executor executor) {
-        this(executor, NoTransactionManager.INSTANCE, NoOpMessageMonitor.INSTANCE);
+    protected AsynchronousCommandBus(Builder builder) {
+        super(builder);
+        this.executor = builder.executor;
     }
 
     /**
-     * Initialize the AsynchronousCommandBus using the given {@code executor}, {@code transactionManager} and
-     * {@code messageMonitor}.
+     * Builder class to instantiate a {@link AsynchronousCommandBus}.
+     * <p>
+     * The {@link TransactionManager} is defaulted to a {@link NoTransactionManager}, the {@link MessageMonitor} is
```

Comment:
Should this information be on both the `builder()` method and the `Builder` class?

`Builder class to instantiate a {@link AsynchronousCommandBus}.` suggests it is a comment for the Builder class and not the method.

Reply:
If it causes confusion, I think it needs adjustment.
Changed to `Instantiate a Builder to be able to create a {@link AsynchronousCommandBus}.`

Additionally, changed this in all the `builder()` functions, also of the previous PR.